src/main/java/svydovets/core/context/beanFactory/BeanFactory.java

```diff
@@ -34,12 +32,20 @@
 public class BeanFactory {
     public static final String NO_BEAN_FOUND_OF_TYPE = "No bean found of type %s";
     private final Map<String, Object> beanMap = new LinkedHashMap<>();
-    private final List<BeanPostProcessor> beanPostProcessors = new ArrayList<>(List.of(new AutowiredAnnotationBeanPostProcessor()));
+    private final List<BeanPostProcessor> beanPostProcessors = new ArrayList<>();
     private final PackageScanner packageScanner = new PackageScanner();
     private final BeanDefinitionFactory beanDefinitionFactory = new BeanDefinitionFactory();
 
+    private CommandFactory commandFactory = new CommandFactory();
+
+    public BeanFactory() {
+        commandFactory.registryCommand(CommandFactoryEnum.FC_GET_BEAN, this::getBean);
+        commandFactory.registryCommand(CommandFactoryEnum.FC_GET_BEANS_OF_TYPE, this::getBeansOfType);
```

Comment:
```suggestion
        commandFactory.registryCommand(CommandFactoryEnum.FC_GET_BEANS_OF_TYPE, this::getBeansByType);
```

Reply:
@IgorZaporozhtsev  this is method reference to BeanFactory, firstly have to be renamed there